Apps/Mock/src/Controllers/PhsaController.cs

```diff
@@ -59,14 +59,29 @@ public ContentResult Covid19Orders([FromQuery] string subjectHdid, [FromQuery] s
             return new ContentResult { Content = payload!, ContentType = "application/json" };
         }
 
+        /// <summary>
+        /// Gets mock data for COVID-19 reports.
+        /// </summary>
+        /// <param name="id">The laboratory order ID.</param>
+        /// <param name="subjectHdid">The HDID of the patient.</param>
+        /// <returns>The mocked COVID-19 report json.</returns>
+        [HttpGet]
+        [Route("laboratoryOrder/{id}/LabReportDocument")]
```

Comment:
This route should be `"Covid19Orders/{id}/LabReportDocument"` to match the change in appsettings.